src/claptools.rs

```diff
@@ -29,6 +29,9 @@ pub struct ListenerAddressPositional {
     ///
     /// * UNIX socket path like /tmp/mysock or Linux abstract address like @abstract
     ///
+    /// * AF_VSOCK socket in format vsock:cid:port, like vsock:42:8080
+    ///   Refer vsock manual for details: https://www.man7.org/linux/man-pages/man7/vsock.7.html
```

Comment:
Maybe you meant more spaces before `Refer` for it to render nicely.

Here how it currently looks:

![doc](https://github.com/user-attachments/assets/c852a11a-96b1-4dfc-8ffb-46ff0552487c)

I expect you intended it to look other way.

If you run `cargo doc --all-features` you also see some warning regarding the unformatted link. Note that there are currently a lot of other, unrelated doc warnings.

Reply:
Here probably should be a dot before "refer" word, but it easily could be mistaken as part of syntax

Reply:
Maybe that part is too lengthy for a CLI help message anyway, so it may be worth to keep it like that:

    AF_VSOCK socket in format vsock:cid:port, like vsock:any:8080

There is no reference to e.g. `unix(7)` or `inet(7)` anyway nearby.